concepts/match-basics/.meta/config.json

```
@@ -1,7 +1,5 @@
 {
-    "blurb": "<todo>",
-    "authors": [
-        "misicnenad"
-    ],
-    "contributors": []
+  "blurb": "The `match` expression allows for easy comparison of a value against a series of patterns and afterwards run the code based on which pattern matches.",
+  "authors": ["Ephraim-nonso"],
+  "contributors": []
```

```suggestion
  "contributors": ["misicnenad"]
```
Add me as just a contributor (for the effort of reviewing the PR)